ament_tools/helper.py

```diff
@@ -222,6 +222,10 @@ def extract_argument_group(args, delimiting_option):
     return trimmed_args, extracted_args
 
 
+def compute_deploy_destination(context, filename, dst_subfolder=''):
+    return os.path.join(context.install_space, dst_subfolder, filename)
```

Comment:
This custom function only takes three arguments and passes them to `os.path.join` (except using `install_space`). Does this really have to be a separate function? I don't see a value over using `os.path.join` directly.